maker/maker.go

```diff
@@ -117,6 +148,15 @@ func MakeInterface(comment, pkgName, ifaceName, ifaceComment string, methods []s
 	return FormatCode(code)
 }
 
+// isFunctionPrivate checks whether the starting
+// sign is lower case
+func isFunctionPrivate(name string) bool {
+	if len(name) == 0 {
```

Comment:
this doesn't look like a part of this function, if there is no name, it should not even be called. otherwise, "no name" is treated as a public function.